### PR TITLE
Case 21781: show up audio screen using Settings > Audio

### PR DESCRIPTION
**Pull request by wayne-chen**
# TEST PLAN
- Open Interface in desktop mode
- Open audio settings screen using the menu with `Settings > Audio`
- Screen should be visually/functionally the same as opening through the audio app in the toolbar
- Switch to HMD mode
- Open tablet
- Open audio settings screen by clicking through `Menu > Settings > Audio`
- Screen should be visually/functionally the same as opening through the audio app and desktop mode `Settings > Audio` screen through the menu

https://highfidelity.manuscript.com/f/cases/21781/RC82-Audio-Settings-window-accessed-by-Settings-Audio-is-blank

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18225/HighFidelity-Beta-PR15211-db22fa9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18225/HighFidelity-Beta-Interface-PR15211-db22fa9.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18225/HighFidelity-Beta-PR15211-db22fa9.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18225/HighFidelity-Quest-Beta-PR15211-db22fa9.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18225/HighFidelity-Beta-PR15211-db22fa9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18225/HighFidelity-Beta-Interface-PR15211-db22fa9.exe

File: interface/src/Menu.cpp [hunk 1]
```diff
@@ -270,10 +270,14 @@ Menu::Menu() {
     // Settings > Audio...
     action = addActionToQMenuAndActionHash(settingsMenu, "Audio...");
     connect(action, &QAction::triggered, [] {
-        static const QUrl widgetUrl("hifi/dialogs/Audio.qml");
         static const QUrl tabletUrl("hifi/audio/Audio.qml");
-        static const QString name("AudioDialog");
-        qApp->showDialog(widgetUrl, tabletUrl, name);
+        auto tablet = DependencyManager::get<TabletScriptingInterface>()->getTablet("com.highfidelity.interface.tablet.system");
+        auto hmd = DependencyManager::get<HMDScriptingInterface>();
+        tablet->pushOntoStack(tabletUrl);
```
Comment (danteruiz) [inline interface/src/Menu.cpp]: you can just use Tablet::loadQMLSource

Comment (wayne-chen) [reply interface/src/Menu.cpp]: I was copying the same style as the opening of the other QML files in the file.

**Pull request merged**